lib/std/zig/parse.zig

```diff
@@ -1385,9 +1384,6 @@ const Parser = struct {
                 .keyword_catch => {
                     _ = try p.parsePayload();
                 },
-                .invalid_ampersands => {
-                    try p.warn(.invalid_and);
-                },
```

Comment:
Not very important, but the comment about "&&" above doesn't make sense after this change (I was confused for a bit trying to figure out what it meant). This switch can probably just be an if now.